src/DotLiquid.Tests/FilterTests.cs

```diff
@@ -206,6 +256,36 @@ public void TestFilterWithMultipleMethodSignaturesAndContextParam()
             Assert.AreEqual("ABC", Template.Parse("{{'A' | concat_with_context : 'B', 'C'}}").Render());
         }
 
+        [Test]
+        public void TestFilterWithMultipleMethodSignaturesDifferentClasses()
+        {
+            Template.RegisterFilter(typeof(FiltersWithMulitpleMethodSignaturesDifferentClassesOne));

```

Comment:
Since this is global, the behavior is sort of unpredictable across tests. Should we make a test of filters local to the context.

Reply:
Added context tests here: https://github.com/dotliquid/dotliquid/pull/453/commits/67a6be4bf74ce0ec2a7b86c6110f6d2b9d069523

Reply:
I was referring to localFilters attribute as used [here](https://github.com/dotliquid/dotliquid/blob/master/src/DotLiquid.Tests/ShopifyFiltersTests.cs)

Reply:
Ah, okay! I added some tests to cover that case here: https://github.com/dotliquid/dotliquid/pull/453/commits/31139cdc569c2a28e6ecdc63b36db3f80a7fd12e